projects/addon-doc/components/api/api-item.component.ts

```diff
@@ -82,9 +85,13 @@ export class TuiDocAPIItem<T> implements AfterViewInit {
     }
 
     public emitEvent(event: unknown): void {
+        const alert =
+            event instanceof Object ? tuiInspectAny(event, 2) : (event as string);
+
         this.alerts
-            .open(event ?? tuiInspectAny(event, 2), {label: this.name})
-            .subscribe();
+            .open(alert, {label: this.name})
+            .pipe(takeUntilDestroyed(this.destroyRef))
```

Comment:
We do not need to unsubscribe, it's autoclosing alert - they complete automatically

Reply:
fixed